### PR TITLE
fix(CommandLine): examples

### LINKED ISSUE #40757: Fix dotnet run example in System.CommandLine tutorial
### Type of issue

Code doesn't work

### Description

In the chapter **Test the app**, it say you can test the app with the command `dotnet run -- --file scl.runtimeconfig.json` executed from a shell in the project directory.

But with `dotnet run`, the working directory is the project directory, so the file **scl.runtimeconfig.json** isn't found and the execution fail.

The working command is :
```cmd
dotnet run -- --file bin/Debug/net6.0/scl.runtimeconfig.json/scl.runtimeconfig.json
```

I suggest to modify the command and add a short explanation why this path.
If you agree, I can do the PR.

### Page URL

https://learn.microsoft.com/en-us/dotnet/standard/commandline/get-started-tutorial

### Content source URL

https://github.com/dotnet/docs/blob/main/docs/standard/commandline/get-started-tutorial.md

### Document Version Independent Id

fa160b1d-66ed-8c5d-37cd-b7e02443e9f1

### Article author

@tdykstra

### Metadata

* ID: 7705dc41-7297-0807-7e9c-2e6548c26b84 
* Service: **dotnet-fundamentals**

---
[Associated WorkItem - 287048](https://dev.azure.com/msft-skilling/Content/_workitems/edit/287048)

### PR DESCRIPTION
**Pull request by vernou**
## Summary

- Modify a example to the `async main` await the main taks.
- Fix a example to run the example program

Fixes #40757 (and its duplicate dotnet/command-line-api#2427)

---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [docs/standard/commandline/get-started-tutorial.md](https://github.com/dotnet/docs/blob/d30804cc3e78dc9ce59e572fa677ff074052a4cc/docs/standard/commandline/get-started-tutorial.md) | [Tutorial: Get started with System.CommandLine](https://review.learn.microsoft.com/en-us/dotnet/standard/commandline/get-started-tutorial?branch=pr-en-us-41823) |

### CODE REVIEW

File: docs/standard/commandline/get-started-tutorial.md [hunk 1]
```diff
@@ -93,9 +93,11 @@ You can use any of the following ways to test while developing a command-line ap
 * Use `dotnet run` and pass option values to the app instead of to the `run` command by including them after `--`, as in the following example:
 
   ```dotnetcli
-  dotnet run -- --file scl.runtimeconfig.json
+  dotnet run -- --file bin/Debug/net6.0/scl.runtimeconfig.json
   ```
 
+> The working directory is the csproj folder, so the relative path to `scl.runtimeconfig.json` is from the csproj folder.
```
Comment (tdykstra) [inline docs/standard/commandline/get-started-tutorial.md:99]:
```suggestion
The working directory is the project folder (the folder that has the .csproj  file), so the relative path to `scl.runtimeconfig.json` is from the project folder.
```

**Review by tdykstra — COMMENTED** [review]
Looks good, I left a suggestion for a clarification.

**Review by tdykstra — APPROVED** [review]

**Comment by vernou** [comment]
@tdykstra, what is the next step?

**Comment by tdykstra** [comment]
> @tdykstra, what is the next step?

The next step was for me to merge it, which I just now did. Thanks for contributing this.

**Comment by vernou** [comment]
Ok, you are welcome.

**Pull request merged**